tests/controllers_tests/controllers_test.go

```diff
@@ -747,6 +747,44 @@ func TestRetrievePeopleIdAndCohort(t *testing.T) {
 	}
 }
 
+func TestRetrievePeopleIdAndCohortBadCohortPair(t *testing.T) {
+	cohortId := 1
+	cohortPairs := [][]int{
+		{4, 5},
+	}
+
+	cohortData := []*models.PersonConceptAndValue{
+		{
+			PersonId: 1,
+		},
+		{
+			PersonId: 2,
+		},
+		{
+			PersonId: 3,
+		},
+	}
+
+	expectedResults := map[int64]map[string]string{
+		int64(1): {
+			"ID_4_5": "NA",
+		},
+		int64(2): {
+			"ID_4_5": "NA",
+		},
+		int64(3): {
+			"ID_4_5": "NA",
+		},
+	}
```

Comment:
this is the scenario where persons are not part of any of the cohorts right? What about the scenario where they are part of both cohorts? I.e. just create a cohort pair like
```go
cohortPairs := [][]int{
		{3,3},
	}
```
Other than that is a strange/non-realistic scenario, I think this should return NA as well.